compiler/passes/normalize.cpp

```diff
@@ -1952,18 +1952,66 @@ static void normVarTypeWoutInit(DefExpr* defExpr) {
     var->type = type;
 
   } else {
-    VarSymbol* typeTemp = newTemp("type_tmp");
-    DefExpr*   typeDefn = new DefExpr(typeTemp);
-    CallExpr*  initCall = new CallExpr(PRIM_INIT, typeExpr);
-    CallExpr*  initMove = new CallExpr(PRIM_MOVE, typeTemp, initCall);
+    bool handled = false;
```

Comment:
This is fair - I was planning on taking the outer if statements and condensing them into a single function call at a later stage, but was in prototyping mode so took the quick path.